### PR TITLE
Rename and update the username

### PR DESCRIPTION
**Pull request by YashkShrivas4491**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4460"></a>

### CODE REVIEW

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻 Thankas for your contribution.

**Comment by YashkShrivas4491** [comment]
> LGTM 👍🏻 Thankas for your contribution.

Thanks 🤩🚀

**Review by Hrishi5111998 — COMMENTED** [review]
hey @YashkShrivas4491 you have mistakenly added wrong link to your **github** profile, please remove **.png** extension from your link. thank you!

File: data/YashkShrivas4491.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Yash Kumar Shrivas",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Information Technology Student || Open-Source || Learning DSA || Learning Web Development || Bachelor of Technology (BTech) || Techy Geek",
+  "links": [
+     {
+      "name": "Follow me on Github",
+      "url": "https://github.com/YashkShrivas4491.png",
```
Comment (priyankarpal) [inline data/YashkShrivas4491.json:9]: change the URL

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/YashkShrivas4491.json (hunk 1 above)
Comment (YashkShrivas4491) [reply data/YashkShrivas4491.json:9]: ok

**Comment by YashkShrivas4491** [comment]
> hey @YashkShrivas4491 you have mistakenly added wrong link to your **github** profile, please remove **.png** extension from your link. thank you!

Dove☑️

File: data/YashkShrivas4491.json (hunk 1 above)
Comment (YashkShrivas4491) [reply data/YashkShrivas4491.json]: Done

**Review by priyankarpal — APPROVED** [review]
LGTM

**Review by Hrishi5111998 — DISMISSED** [review]
LGTM

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please revert the changes in the other files and keep your profile changes only.
Let us know if you need any help doing that. 🙂

**Review by priyankarpal — CHANGES_REQUESTED** [review]
hey, @YashkShrivas4491 here is a conflict error please solve it.

**Comment by YashkShrivas4491** [comment]
> Please revert the changes in the other files and keep your profile changes only. Let us know if you need any help doing that. 🙂

I deleted conflicting files , that's all i can do from my side . if any improvement needed , guide me

**Review by priyankarpal — CHANGES_REQUESTED** [review]
hey @YashkShrivas4491 here is a conflicts error you need to solve it. If you don't know you can check this official doc https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github
You can ask here if you're facing any problems. ✌️

**Comment by YashkShrivas4491** [comment]
> hey @YashkShrivas4491 here is a conflicts error you need to solve it. If you don't know you can check this official doc https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github You can ask here if you're facing any problems. ✌️

Can you tell me why this conflicts are occurring

**Comment by priyankarpal** [comment]
> > hey @YashkShrivas4491 here is a conflicts error you need to solve it. If you don't know you can check this official doc https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github You can ask here if you're facing any problems. v
> 
> Can you tell me why this conflicts are occurring

Conflicts in GitHub can occur when two or more users make separate changes to the same line or block of code in a shared repository, and Git is unable to automatically merge these changes. This can lead to a conflict that needs to be resolved manually. Other factors that can contribute to conflicts include conflicting changes made to the same file, different branches being merged, or changes to files being pushed simultaneously. It's important to regularly pull updates from the repository and communicate with other contributors to prevent conflicts from occurring.

**Comment by YashkShrivas4491** [comment]
> > > hey @YashkShrivas4491 here is a conflicts error you need to solve it. If you don't know you can check this official doc https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github You can ask here if you're facing any problems. v
> > 
> > 
> > Can you tell me why this conflicts are occurring
> 
> Conflicts in GitHub can occur when two or more users make separate changes to the same line or block of code in a shared repository, and Git is unable to automatically merge these changes. This can lead to a conflict that needs to be resolved manually. Other factors that can contribute to conflicts include conflicting changes made to the same file, different branches being merged, or changes to files being pushed simultaneously. It's important to regularly pull updates from the repository and communicate with other contributors to prevent conflicts from occurring.

sure ☑️

**Comment by eddiejaoude** [comment]
Wow a lot of collaboration going on here which is great 👍 

But unfortunately, deleting the files is not correct as these files are needed, you should remove the changes from your pull request not delete the files as these are still changes

I think it is best, if I close this pull request and you follow the quickstart guide and create a new pull request https://linkfree.eddiehub.io/docs/quickstart

**Comment by YashkShrivas4491** [comment]
> Wow a lot of collaboration going on here which is great 👍
> 
> But unfortunately, deleting the files is not correct as these files are needed, you should remove the changes from your pull request not delete the files as these are still changes
> 
> I think it is best, if I close this pull request and you follow the quickstart guide and create a new pull request https://linkfree.eddiehub.io/docs/quickstart

yup Sure I will do it

**Pull request closed without merging**